src/uwtools/apps/srw.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python3
+#pylint: disable=unused-import, unused-variable, unused-argument, useless-parent-delegation
+# remove these disables once implemented
+'''
+This file contains the specific drivers for a particular app, using the facade pattern base class.
+'''
+
+import logging
+import os
+import sys
+import shutil
+import subprocess
+
+from uwtools.utils import file_helpers
+from uwtools.utils import cli_helpers
+from ..drivers.facade import Facade
+
+logging.getLogger(__name__)
```

Comment:
I don't think this line or the two copies in other classes actually do anything for us: They return a reference to the current logger, but we don't save it. The docs (https://docs.python.org/3/library/logging.html#logging.getLogger) don't mention any side effects, so I think these lines are no-ops. I'd be happy to be mistaken.

Reply:
Hmm - PR this came from is #188 There are a few other places where we instantiate the logger object with a name based on the first creating class, then calling that object back as needed to prevent too many objects floating and if we need its attributes. We have refactored this a bit though, I'll have to think on this again in the morning to make sure its still doing what we intend to